docs/source/main_classes/logging.mdx

```diff
@@ -54,6 +54,8 @@ verbose to the most verbose), those levels (with their corresponding int values
 - `transformers.logging.INFO` (int value, 20): reports error, warnings and basic information.
 - `transformers.logging.DEBUG` (int value, 10): report all information.
 
+By default, `tqdm` progress bars will be displayed to show progress during model download, training and evaluation in the PyTorch [`Trainer`] and various other places. [`transformers.utils.tqdm_utils.set_progress_bar_enabled`] can be used to suppress or unsuppress this behavior. 
```

Comment:
`set_progress_bar_enabled` does not mean anything to me. I understand it's been taken from datasets so we may want to keep it for consistency between the libraries but can we add `disable_progress_bars()` and `enable_progress_bars()` as convenience methods with better names?

Reply:
I agree that the naming might not be as intuitive. Perhaps worthy of note, however, is that `disable_progress_bar()` is a [deprecated pattern](https://github.com/huggingface/datasets/blob/ccc28d4141006924af85c2d940d82d71da4c6ec0/src/datasets/utils/tqdm_utils.py#L83-L95) in `datasets`. 

```
@deprecated("Use set_progress_bar_enabled(False) instead.")
def disable_progress_bar():
    """Disable tqdm progress bar.

    .. deprecated:: 1.12.2
        Use set_progress_bar_enabled(False) instead.
```

Part of me is concerned that this might bring further confusion, but if you think this is a non-issue, I'll go ahead and add the two functions!

Reply:
@lhoestq Why is this pattern deprecated?

Reply:
It comes from this PR: https://github.com/huggingface/datasets/pull/3031

Maybe let's discuss what should be the best API for `transformers` and `datasets` for consistency ? also cc @mariosasko

Reply:
I personally would prefer having the two methods mentioned above.

Reply:
As explained in the linked PR, my idea was to align the naming with `set_caching_enabled/is_caching_enabled` in `datasets`.

@lhoestq   
I guess we can add `enable_progress_bars` and remove the deprecation notice from `disable_progress_bar`, but then we should also do the same for the caching mechanism? WDYT? Also, I'm not a big fan of having two ways to do the same thing (`set_progress_bar_enabled(True)` and `enable_progress_bars()`).

Reply:
Perhaps `tqdm_progress_bars_toggle(True/False)` 

I added `tqdm` as since if it's always tqdm that is being used then it's really tqdm that we fiddle with.

but `progress_bars_toggle(True/False)`  is fine too.

Reply:
I'm fine with one method if it's better named, for instance the last suggestion of @stas00 woks for me too. It's just that`set_progress_bar_enabled` is very hard to parse IMO.

Reply:
thinking more, linguistically "toggle" means to switch to the opposite state, so the name I proposed is a not the best.

Reply:
Perhaps having these 2 is the most intuitive? `tqdm_progress_bars_enable()` and `tqdm_progress_bars_disable()`

Reply:
I'm fine if we go back to using the `disable_progress_bar` in `datasets`.

Note that is doesn't have an `s` in the end though, contrary to your first suggestion @sgugger - so I would prefer if the one in `transformers` doesn't have an `s` either.

Then in this case @mariosasko we should have the `enable_progress_bar` function as well, and probably align the corresponding methods for caching indeed

Reply:
Works for me!